setup.cfg

```diff
@@ -21,6 +21,9 @@ classifiers =
 [options]
 packages = sevsnpmeasure
 python_requires = >=3.7
+install_requires = 
+    cryptography==39.0.1
+    types-cryptography
 
 [options.entry_points]
 console_scripts = sev-snp-measure = sevsnpmeasure.cli:main
```

Comment:
we need to figure out how to add another console script here (the new `snp-create-id-block` script introduced in this PR).